src/routes.js

```diff
@@ -1,12 +1,14 @@
 import userRouter from './routes/user';
+import emailRouter from './routes/email';
 import userAuthRouter from './routes/user/auth';
 
-const makeRoutes = app => {
+const routes = app => {
   // public routes
   app.use('/public/v1/users', userRouter);
+  app.use('/public/v1', emailRouter);
```

Comment:
This should be registered under /auth because only authenticated users should be able to access it.

Reply:
done